src/util.rs

```diff
@@ -17,3 +17,10 @@ pub fn zero(slice: &mut [f32]) {
         *sample = 0.0;
     }
 }
+
+
+pub type SampleSeq = u64;
+pub type Ms = f64;
+pub fn sample_seq_duration_ms(sample_rate: usize, first: SampleSeq, last: SampleSeq) -> Ms {
+    (last - first) as f64 / sample_rate as f64 * 1000.0
+}
```

Comment:
Also interested in any feedback here

Reply:
I think you have to put the output into the input if you want feedback 🥁

Reply:
What do you think of changing this function to return a value in seconds? Mixlab mostly uses seconds or samples as timing units, and I notice that in Clock we're dividing this value by 1000.0 before using it anyway